data/schema.js

```diff
@@ -1,9 +1,19 @@
 export default [`
+type Subscriber {
+  email: String
+}
+
 type Query {
-  foo: String
+  subscriber (email: String!): Subscriber
+  allSubscribers: [Subscriber]
+}
```

Comment:
don't think queries are needed? and re: second one, don't want anyone to be able to get my list of subscribers